PerimeterXModule/Internals/CredentialsIntelligence/LoginSuccessful/CustomLoginSuccessfulParser.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Web;
+using PerimeterX.CustomBehavior;
+
+namespace PerimeterX
+{
+    public class CustomLoginSuccessfulParser : ILoginSuccessfulParser
+    {
+        ILoginSuccessfulHandler loginSuccessfulParserhandler;
+
+        public CustomLoginSuccessfulParser(PxModuleConfigurationSection config) 
+        {
+            loginSuccessfulParserhandler = PxCustomFunctions.GetCustomLoginSuccessfulHandler(config.CustomLoginSuccessfulHandler);
+        }
+
+        public bool IsLoginSuccessful(HttpResponse httpResponse)
+        {
+            try
+            {
+                if (loginSuccessfulParserhandler != null)
+                {
+                    return loginSuccessfulParserhandler.Handle(httpResponse);
+                } 
+            }
+            catch (Exception ex)
+            {
+                PxLoggingUtils.LogDebug("An error occurred while executing login successful handler " + ex.Message);
+                
+            }
+
+            return false;
```

Comment:
This should return `null` instead of false. We want to be 100% sure that true and false are accurate. If we aren't sure, it should be `null`.